CMakeLists.txt

```diff
@@ -2,5 +2,6 @@ cmake_minimum_required(VERSION 3.9)
 project(stocklion)
 
 set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_BUILD_TYPE Debug)
```

Comment:
Going to remove this, thought I needed it for debugging but my hosts file was causing issues.